src/user/tipsandtricks.rst

```diff
@@ -152,3 +152,36 @@ In order to override the default behavior, a user can set the environment variab
    Note that you should select the cudatoolkit version most appropraite for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. (At the time of writing, there seems to be a bug in how the cuda builds are resolved by mamba, defaulting to cudatoolkit==10.2; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above.) 
 
 For context, installing TensorFlow 2.7.0 with ``CONDA_CUDA_OVERRIDE="11.2" mamba install "tensorflow==2.7.0=cuda*" -c conda-forge`` results in approximately 2 GB of packages to download while ``CONDA_CUDA_OVERRIDE="11.2" mamba install "tensorflow=2.7.0=cpu*" -c conda-forge`` results in approximately 200 MB to download. That is a significant bandwidth and storage wasted if one only needs the ``-cpu`` variant! 
+
+
+.. _pypy:
+
+Using PyPy as an interpreter
+============================
+The ``conda-forge`` channel supports creating and installing packages into
+environments using the `PyPy interpreter`_. Many packages are already
+available. You need to enable the ``conda-forge`` channel and use
+the ``pypy`` identifier when creating your environment:
+
+.. code-block:: shell
+
+    $ conda create -c conda-forge -n my-pypy-env pypy python=3.8
```

Comment:
@isuruf I moved the directions to here since it seems more user-facing than maintainer-facing. This way I can also link to here as the canonical directions for installation.